ibmcloud/README.md

```diff
@@ -136,7 +136,7 @@ This Terraform configuration also triggers execution of an Ansible playbook to s
 If ansible fails for some reason, you can rerun the ansible playbook as follows.
```

Comment:
Small note but should this been updated to `ansible playbooks`?

Reply:
Thanks, made both changes.

Reply:
LGTM